scss/abstracts/variables/_typography.scss

```diff
@@ -0,0 +1,68 @@
+$base: 18px; // used to set rem to 18px
+
+$line-height-regular: 1.6;
+
+$font-weight-regular: 400;
+$font-weight-bold: 600;
+
+// ---------------------------------------------------- Font sizes -----------------------------------------------------
+$type-matrix: (
+        dp-fs-xxxl: (
+                small: 32px,
+                large: 48px,
+                weight: $font-weight-bold,
+                line-height: 1.3
+        ),
+        dp-fs-xxl: (
+                small: 28px,
+                large: 36px,
+                weight: $font-weight-bold,
+                line-height: 1.4
+        ),
+        dp-fs-xl: (
+                small: 26px,
+                large: 30px,
+                weight: $font-weight-bold,
+                line-height: 1.4
+        ),
+        dp-fs-l: (
+                small: 24px,
+                large: 26px,
+                weight: $font-weight-bold,
+                line-height: 1.4
+        ),
+        dp-fs-m: (
+                small: 20px,
+                large: 22px,
+                weight: $font-weight-bold,
+                line-height: 1.4
+        ),
+        dp-fs-r--b: (
+                small: $base,
+                large: $base,
+                weight: $font-weight-bold,
+                line-height: 1.4
+        ),
+        dp-fs-r: (
```

Comment:
Just to clarify, is the `r` to signify `regular`?

Reply:
Yer you got it, I think in the design guide maybe we can have references to what the shorthand initials mean, or if preferred I could add a comment in code, to save people having to look around.

Reply:
Yeah, that's a good shout 😃